### ISSUE TITLE
Add a11y lint rules to CI

### ISSUE THREAD

**Issue by vic-Gray** [code-flexing/Harvest-Finance#831]
Hint: eslint jsx-a11y in frontend CI; block new violations.

Labels: ux, frontend
